Dune bug??

I have recenetly (a couple of days ago) upgraded my box debian 11. I have normal debian package of dune installed - version: 2.7.1. When doing a clean build of opm-grid I get:
```
In file included from /home/hove/work/OPM/opm-grid/opm/grid/CpGrid.hpp:54,
                 from /home/hove/work/OPM/opm-grid/tests/cpgrid/distribution_test.cpp:9:
/usr/include/dune/common/parallel/variablesizecommunicator.hh: In instantiation of ‘void Dune::VariableSizeCommunicator<Allocator>::communicate(DataHandle&) [with bool forward = true; DataHandle = CheckGlobalCellHandle; Allocator = std::allocator<std::pair<Dune::InterfaceInformation, Dune::InterfaceInformation> >]’:
/usr/include/dune/common/parallel/variablesizecommunicator.hh:443:22:   required from ‘void Dune::VariableSizeCommunicator<Allocator>::forward(DataHandle&) [with DataHandle = CheckGlobalCellHandle; Allocator = std::allocator<std::pair<Dune::InterfaceInformation, Dune::InterfaceInformation> >]’
/home/hove/work/OPM/opm-grid/tests/cpgrid/distribution_test.cpp:728:47:   required from here
/usr/include/dune/common/parallel/variablesizecommunicator.hh:1216:13: error: ‘class CheckGlobalCellHandle’ has no member named ‘fixedsize’; did you mean ‘fixedSize’?
 1216 |   if(handle.fixedsize())
      |      ~~~~~~~^~~~~~~~~
      |      fixedSize
/usr/include/dune/common/parallel/variablesizecommunicator.hh: In instantiation of ‘void Dune::VariableSizeCommunicator<Allocator>::communicate(DataHandle&) [with bool forward = false; DataHandle = CheckGlobalCellHandle; Allocator = std::allocator<std::pair<Dune::InterfaceInformation, Dune::InterfaceInformation> >]’:
/usr/include/dune/common/parallel/variablesizecommunicator.hh:468:23:   required from ‘void Dune::VariableSizeCommunicator<Allocator>::backward(DataHandle&) [with DataHandle = CheckGlobalCellHandle; Allocator = std::allocator<std::pair<Dune::InterfaceInformation, Dune::InterfaceInformation> >]’
/home/hove/work/OPM/opm-grid/tests/cpgrid/distribution_test.cpp:729:47:   required from here
/usr/include/dune/common/parallel/variablesizecommunicator.hh:1216:13: error: ‘class CheckGlobalCellHandle’ has no member named ‘fixedsize’; did you mean ‘fixedSize’?
 1216 |   if(handle.fixedsize())
      |      ~~~~~~~^~~~~~~~~
      |      fixedSize
/usr/include/dune/common/parallel/variablesizecommunicator.hh: In instantiation of ‘void Dune::VariableSizeCommunicator<Allocator>::communicate(DataHandle&) [with bool forward = true; DataHandle = CheckBoundaryIdHandle; Allocator = std::allocator<std::pair<Dune::InterfaceInformation, Dune::InterfaceInformation> >]’:
/usr/include/dune/common/parallel/variablesizecommunicator.hh:443:22:   required from ‘void Dune::VariableSizeCommunicator<Allocator>::forward(DataHandle&) [with DataHandle = CheckBoundaryIdHandle; Allocator = std::allocator<std::pair<Dune::InterfaceInformation, Dune::InterfaceInformation> >]’
/home/hove/work/OPM/opm-grid/tests/cpgrid/distribution_test.cpp:730:43:   required from here
/usr/include/dune/common/parallel/variablesizecommunicator.hh:1216:13: error: ‘class CheckBoundaryIdHandle’ has no member named ‘fixedsize’; did you mean ‘fixedSize’?
 1216 |   if(handle.fixedsize())
      |      ~~~~~~~^~~~~~~~~
      |      fixedSize
make[2]: *** [CMakeFiles/distribution_test.dir/build.make:82: CMakeFiles/distribution_test.dir/tests/cpgrid/distribution_test.cpp.o] Error 1
```

--- a/tests/cpgrid/distribution_test.cpp
+++ b/tests/cpgrid/distribution_test.cpp
@@ -120,7 +120,7 @@ public:
 
     typedef int DataType;
 
-#if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 7)
+#if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 8)
     bool fixedSize()
 #else
     bool fixedsize()
@@ -221,7 +221,7 @@ public:
     {}
 
     typedef int DataType;
-#if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 7)
+#if DUNE_VERSION_NEWER(DUNE_COMMON, 2, 8)
     bool fixedSize()
 #else
     bool fixedsize()

